MobiFlight/Joysticks/Joystick.cs

```diff
@@ -157,7 +121,7 @@ protected virtual void EnumerateDevices()
                         Log.Instance.log($"Axis can't be mapped: {DIJoystick.Information.InstanceName} Aspect: {aspect} Offset: {offset} Usage: {usage} Axis: {name} Label: {axisLabel}.", LogSeverity.Error);
                         continue;
                     }
-                    Axes.Add(new JoystickDevice() { Name = axisName, Label = axisLabel, Type = JoystickDeviceType.Axis });
+                    Axes.Add(new JoystickDevice() { Name = axisName, Label = axisLabel, Type = DeviceType.AnalogInput });
```

Comment:
I don't like that we lose the specifics of a Joystick. An axis inherits from Analog Input maybe. It is an Enum right now, i know. but I find it sad that we use the base types here. It could be a AxisJoystickDevice.

Reply:
Added extra JoystickDeviceType.